oneflow/api/python/framework/tensor_functions.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <Python.h>
+#include "oneflow/api/python/exception/exception.h"
+#include "oneflow/api/python/functional/common.h"
+#include "oneflow/api/python/functional/functional_api.yaml.pybind.h"
+#include "oneflow/core/functional/functional.h"
+
+namespace oneflow {
+namespace one {
+
+using functional::PyObjectPtr;
+
+#define ASSERT(x) (x).GetOrThrow()
+#define ASSERT_PTR(x) (x).GetPtrOrThrow()
+
+static PyObject* PyTensorObject_reshape(PyObject* self, PyObject* args) {
+  HANDLE_ERRORS
+  PyObject* new_shape = NULL;
+  if (PyTuple_Size(args) == 1) {
+    new_shape = PyTuple_GetItem(args, 0);
+    if (PyLong_Check(new_shape)) new_shape = PyTuple_Pack(1, new_shape);
+  } else {
+    new_shape = args;
+  }
+  PyObject* tuple = PyTuple_Pack(2, self, new_shape);
+  return functional::reshape(NULL, tuple, NULL);
+  Py_DECREF(new_shape);
+  Py_DECREF(tuple);
+  END_HANDLE_ERRORS
+}
+
+static PyObject* PyTensorObject_reshape_as(PyObject* self, PyObject* args, PyObject* kwargs) {
+  HANDLE_ERRORS
+  auto tensor = PyTensor_Unpack(self);
+  PyObject* other = NULL;
+  static const char* keywords[2] = {"other", NULL};
+  if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|O:other", const_cast<char**>(keywords),
```

Comment:
```suggestion
  if (!PyArg_ParseTupleAndKeywords(args, kwargs, "O|:reshape_as", const_cast<char**>(keywords),
```
PyArg_ParseTupleAndKeywords里面这个字符串中，冒号后面是报错时候显示的函数名；other不是可选参数，所以前面的flag应该是"O|"，"|"后面表示optional。

Reply:
好的，已修改